src/main/java/frc/robot/FieldConstants.java

```diff
@@ -125,10 +122,11 @@ public static final class Stage {
   public static final AprilTagFieldLayout aprilTags;
 
   static {
-    try {
-      aprilTags = AprilTagFieldLayout.loadFromResource(k2024Crescendo.m_resourceFile);
-    } catch (IOException e) {
-      throw new RuntimeException(e);
-    }
+    //    try {
+    // aprilTags = AprilTagFieldLayout.loadFromResource(k2024Crescendo.m_resourceFile);
+    aprilTags = null;
```

Comment:
isn't this the same as never initializing it?

Reply:
that's the whole point